src/bot.js

```diff
@@ -1,33 +1,36 @@
+"use strict";
 
-'use strict'
+const slack = require("slack");
+const _ = require("lodash");
+const config = require("./config");
 
-const slack = require('slack')
-const _ = require('lodash')
-const config = require('./config')
-
-let bot = slack.rtm.client()
+let bot = slack.rtm.client();
 
 bot.started((payload) => {
-  this.self = payload.self
-})
+  this.self = payload.self;
+});
 
 bot.message((msg) => {
-  if (!msg.user) return
-  if (!_.includes(msg.text.match(/<@([A-Z0-9])+>/igm), `<@${this.self.id}>`)) return
-
-  slack.chat.postMessage({
-    token: config('SLACK_TOKEN'),
-    icon_emoji: config('ICON_EMOJI'),
-    channel: msg.channel,
-    username: 'Starbot',
-    text: `beep boop: I hear you loud and clear!"`
-  }, (err, data) => {
-    if (err) throw err
-
-    let txt = _.truncate(data.message.text)
-
-    console.log(`🤖  beep boop: I responded with "${txt}"`)
-  })
-})
-
-module.exports = bot
+  if (!msg.user) return;
+  if (!_.includes(msg.text.match(/<@([A-Z0-9])+>/gim), `<@${this.self.id}>`))
+    return;
+
+  slack.chat.postMessage(
+    {
+      token: config("SLACK_TOKEN"),
+      icon_emoji: config("ICON_EMOJI"),
+      channel: msg.channel,
+      username: "Lighthouse",
```

Comment:
Updated this to `Lighthouse`.

Reply:
Not specifically related to this change but currently `config("SLACK_TOKEN")` returns undefined since we're still yet to define it in the `.env` file (which we'd have to add in `.gitignore` btw)!